lib/Doctrine/DBAL/Connection.php

```diff
@@ -1374,6 +1433,12 @@ public function project($sql, array $params, Closure $function)
      */
     public function query()
     {
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/pull/4163',
+            'Connection::query() is deprecated, use Connection::executeQuery() instead.'
```

Comment:
I think it's okay to omit the class name in `executeQuery()` since it's the same class that contains the deprecated method. It would be consistent with the other messages.